src/main/java/net/mcreator/element/types/Tag.java

```diff
@@ -48,6 +48,11 @@ public String tagType() {
 		return type.toLowerCase(Locale.ENGLISH);
 	}
 
+	//TODO: Temporal, used for testing

```

Comment:
Remove this then?

Reply:
Since item properties PR is too big for this feature to be merged into it, I made this to test list templates separately. This will be removed when you consider everything works correctly.

Reply:
But tags actually use this, so how is this temporal. It is not clear to me

Reply:
This is just a setup for you/anyone else to test the feature, I could remove this if you already tested tags generation with list templates.

Reply:
Ok, so once I test it and do first approval, you plan to remove this? I probably misunderstood your previous comment then :D